src/components/molecules/Common/AssetModal/AssetContainer/index.tsx

```diff
@@ -23,22 +23,26 @@ export type Props = {
   assets?: Asset[];
   isMultipleSelectable?: boolean;
   accept?: string;
-  onCreateAsset?: (file: File) => void;
+  onCreateAsset?: (files: FileList) => void;
+  onRemove?: (assets: string[]) => void;
```

Comment:
Sorry I should have suggested earlier, but just to make it very clear naming the passed value assetIds would be best I think. We aren't passing assets so it isn't quite correct.